registry/Cargo.toml

```diff
@@ -16,6 +17,36 @@ tracing-subscriber = "0.3.17"
 sqlx = { version = "0.7.2", features = ["runtime-tokio", "postgres", "macros", "migrate"] }
 eyre = "0.6.8"
 url = "2.4.1"
+axum_tonic = "0.1.0"
```

Comment:
Super nit picky / housekeeping related, but could we eventually sort the deps? I can also do it after the merge

Reply:
Of course! By the way, if this is something we care about, we can even enforce it in CI with [cargo sort](https://crates.io/crates/cargo-sort/1.0.5).